src/test/java/move/ChaMoveBehaviorTest.java

```diff
@@ -0,0 +1,62 @@
+package move;
+
+import java.util.ArrayList;
+import java.util.List;
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.Test;
+import piece.Piece;
+import piece.PieceType;
+import piece.Pieces;
+import piece.Position;
+import piece.Route;
+import piece.Team;
+
+public class ChaMoveBehaviorTest {
```

Comment:
```suggestion
import piece.Team;

class ChaMoveBehaviorTest {
```
class 선언이 public인 것과 섞여있는 것 같네요.